resources/views/home.blade.php

```diff
@@ -1,23 +1,31 @@
 @extends('layouts.app')
-
 @section('content')
-<div class="container">
-    <div class="row">
-        <div class="col-md-8 col-md-offset-2">
-            <div class="panel panel-default">
-                <div class="panel-heading">Dashboard</div>
-
-                @if(Auth::user()->is_admin == 1)
-                  <div class="panel-body">
-                      This is the admin dashboard!
-                  </div>
-                @else
-                  <div class="panel-body">
-                      This is the customer dashboard!
-                  </div>
-                @endif
-            </div>
-        </div>
+@if(Auth::user()->is_admin == 1)
+  <div class="container">
+    <ul class="nav nav-tabs col-md-offset-1 col-md-9">
+      <li class="active"><a href="#tab1" data-toggle="tab">Products</a></li>
+      <li><a href="#tab2" data-toggle="tab">Orders</a></li>
+    </ul>
+    <div class="tab-content col-md-offset-1 col-md-9" id="TabContents">
+      <div class="tab-pane fade active in" id="tab1">
+        <a class="btn btn-primary margin-10" type="button" href="/products/create">Add Product</a>
+        @include("partials/products", ["products" => App\Product::all(), "dashboard" => true])
+      </div>
+      <div class="tab-pane fade" id="tab2">
+        @include("partials/orders", ["orders" => App\Order::all()])
+      </div>
     </div>
-</div>
+  </div>
```

Comment:
I feel that for users, slotting the order page here would be convenient once it's finished. Leave as-is for now.